From 7c4ea106db37ff1979a6ad9b2cf526b4a276eceb Mon Sep 17 00:00:00 2001
From: nabijaczleweli <nabijaczleweli@gmail.com>
Date: Tue, 14 Jan 2020 06:09:34 +0100
Subject: [PATCH 1/2] Don't pad CPU usage to 2 characters

---
 src/print_cpu_usage.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/print_cpu_usage.c b/src/print_cpu_usage.c
index 979e082..3a0bb35 100644
--- a/src/print_cpu_usage.c
+++ b/src/print_cpu_usage.c
@@ -177,7 +177,7 @@ void print_cpu_usage(yajl_gen json_gen, char *buffer, const char *format, const
             *(outwalk++) = *walk;
 
         } else if (BEGINS_WITH(walk + 1, "usage")) {
-            outwalk += sprintf(outwalk, "%02d%s", diff_usage, pct_mark);
+            outwalk += sprintf(outwalk, "%d%s", diff_usage, pct_mark);
             walk += strlen("usage");
         }
 #if defined(__linux__)
@@ -192,7 +192,7 @@ void print_cpu_usage(yajl_gen json_gen, char *buffer, const char *format, const
                 int cpu_diff_idle = curr_cpus[number].idle - prev_cpus[number].idle;
                 int cpu_diff_total = curr_cpus[number].total - prev_cpus[number].total;
                 int cpu_diff_usage = (cpu_diff_total ? (1000 * (cpu_diff_total - cpu_diff_idle) / cpu_diff_total + 5) / 10 : 0);
-                outwalk += sprintf(outwalk, "%02d%s", cpu_diff_usage, pct_mark);
+                outwalk += sprintf(outwalk, "%d%s", cpu_diff_usage, pct_mark);
             }
             int padding = 1;
             int step = 10;
-- 
2.25.1

